test/xdr/void_test.exs

```diff
@@ -5,6 +5,17 @@ defmodule XDR.VoidTest do
   alias XDR.Error.Void, as: VoidErr
 
   describe "Encoding void to binary" do
+    test "when not receive a void struct" do
+      try do
```

Comment:
@FranciscoMolina02 This is an antipattern. Please check and fix all the instances that perform in this way.
https://hexdocs.pm/elixir/master/library-guidelines.html#anti-patterns